src/libraries/System.Text.Json/gen/JsonSourceGenerator.Emitter.cs

```diff
@@ -1161,7 +1177,7 @@ private static string GenerateForType(TypeGenerationSpec typeMetadata, string me
     {typeInfoPropertyTypeRef}? {JsonTypeInfoReturnValueLocalVariableName} = null;
     {WrapWithCheckForCustomConverter(metadataInitSource, typeCompilableName)}
 
-    { /* NB OriginatingResolver should be the last property set by the source generator. */ ""}
+    // OriginatingResolver should be the last property set by the source generator.
```

Comment:
Note that the previous form was intentional. We don't want this comment to be emitted in the source generated code.

Reply:
why not? I think it's useful for users looking at the code same as for us

Reply:
I don't think it matters for people not in the process of editing the code.

Reply:
will be reverted in the next iteration